src/apps/external_executor/protobuf/http.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
```

Comment:
Why is this file required?

Reply:
So that the `EndTx` gRPC message can describe a HTTP response.